.github/workflows/awie.yml

```
@@ -60,21 +60,56 @@ jobs:
             image: packaging-bullseye
             distrib: bullseye
 
-    uses: ./.github/workflows/package.yml
-    with:
-      base_directory: centreon-awie
-      spec_file: centreon-awie/packaging/centreon-awie.spectemplate
-      package_extension: ${{ matrix.package_extension }}
-      image_name: ${{ matrix.image }}
-      module_name: awie
-      major_version: ${{ needs.get-version.outputs.major_version }}
-      minor_version: ${{ needs.get-version.outputs.minor_version }}
-      release: ${{ needs.get-version.outputs.release }}
-      commit_hash: ${{ github.sha }}
-      cache_key: ${{ github.sha }}-${{ github.run_id }}-${{ matrix.package_extension }}-${{ matrix.distrib }}
-    secrets:
-      registry_username: ${{ secrets.DOCKER_REGISTRY_ID }}
-      registry_password: ${{ secrets.DOCKER_REGISTRY_PASSWD }}
+    runs-on: ubuntu-22.04
+
+    container:
+      image: ${{ vars.DOCKER_INTERNAL_REGISTRY_URL }}/${{ matrix.image }}:${{ needs.get-version.outputs.major_version }}
+      credentials:
+        username: ${{ secrets.DOCKER_REGISTRY_ID }}
+        password: ${{ secrets.DOCKER_REGISTRY_PASSWD }}
+
+    name: package ${{ matrix.distrib }}
+
+    steps:
+      - name: Checkout sources
+        uses: actions/checkout@v4
+
+      - name: Find and replace patterns in files
+        run: |
+          find . -type f \
+            -exec grep -qE '(@CENTREON_ETC@)|(@BIN_DIR@)|(CENTREON_DIR)|(CENTREON_LOG)|(CENTREON_VARLIB)|(MODULE_NAME)|(DB_CENTSTORAGE)' {} ';' \
+            -exec sed -i -e "s|@CENTREON_ETC@|${centreon_etc}|g" \
+                          -e "s|@BIN_DIR@|/usr/bin|g" \
+                          -e "s|@CENTREON_DIR@|${centreon_dir}|g" \
+                          -e "s|@CENTREON_LOG@|${centreon_log}|g" \
+                          -e "s|@CENTREON_VARLIB@|${centreon_varlib}|g" \
+                          -e "s|@MODULE_NAME@|centreon-awie|g" \
+                          -e "s|@DB_CENTSTORAGE@|centreon_storage|g" \
+                      {} ';'
+        shell: bash
+
+      - name: Package
+        uses: ./.github/actions/package-nfpm
+        with:
+          nfpm_file_pattern: "centreon-awie/packaging/*.yaml"
+          distrib: ${{ matrix.distrib }}
+          package_extension: ${{ matrix.package_extension }}
+          major_version: ${{ needs.get-version.outputs.major_version }}
+          minor_version: ${{ needs.get-version.outputs.minor_version }}
+          release: ${{ needs.get-version.outputs.release }}
+          arch: all
+          commit_hash: ${{ github.sha }}
+          cache_key: cache-${{ github.sha }}-${{ matrix.package_extension}}-centreon-awie-${{ matrix.distrib }}-${{ github.head_ref || github.ref_name }}
```

the cache key must be the same than the deliver jobs
```suggestion
          cache_key: ${{ github.sha }}-${{ github.run_id }}-rpm-${{ matrix.distrib }}
```